seedlet/assets/sass/blocks/cover/_editor.scss

```diff
@@ -12,7 +12,7 @@
 	.block-editor-block-list__block {
 		color: currentColor; // uses text color specified with background-color options in /blocks/utilities/_style.scss
 
-		a:not(.has-text-color) {
+		a:not(.has-text-color):not(.wp-block-button__link) {
```

Comment:
This rule isn't strictly necessary since Gutenberg's inline color rules will override it. But I think it's worth having to keep in sync with the front end (and to prepare for if/when Gutenberg drops that inlining in the editor).